Documentation.docc/Documentation.md

```diff
@@ -0,0 +1,44 @@
+# ``RevenueCat``
```

Comment:
the stuff in this file is a mixture of our readme and our home page. 
I just wanted to add a quick thing, we can evolve from here. 

Some of the format is different because this doesn't support `<h3>` and other tags. It still looks pretty neat, though.